### DISCUSSION TITLE
audiobooks: out of scope features (developer documentation)

### DISCUSSION THREAD

**Discussion by danielweck** [edrlab/thorium-reader#1753] [Ideas]
For example:

* multiple bookmarks per audio track (?)
* display track number in addition to title?
* sleep timer (because desktop app, not mobile?)
* volume control (because system-wide control is enough?)
* background audio / system tray (?)
* handling concurrent audio playback, multiple reader windows (?)
* audio spectrum animations, etc. (cover image is enough?)
* what else?

Follow-up to PR https://github.com/readium/readium-desktop/pull/938

**Comment by llemeurfr** [comment]
My take:
multiple bookmarks per audio track  -> this is needed IMO
display track number in addition to title -> not needed indeed
sleep timer (because desktop app, not mobile) -> this will be needed IMO, as there is less an less difference btw a mobile and desktop app. 
volume control (because system-wide control is enough) -> not needed indeed
background audio / system tray  -> audio goes on on Mac when the app is minified. Isn't it the same on Windows?
handling concurrent audio playback, multiple reader windows -> not needed indeed
audio spectrum animations, etc. (cover image is enough) -> not needed indeed
